scrapers_next/in/people.py

```diff
@@ -20,9 +20,19 @@ def process_page(self):
         phones = (
             CSS("div .fusion-text.fusion-text-2 p").match(self.root)[1].text_content()
         )
-        phone1, phone2 = re.search(
-            r"Phone:\s(\d{3}-\d{3}-\d{4})\s\|\s(.+)", phones
-        ).groups()
+
+        try:
```

Comment:
Maybe something akin to:
```
phone_numbers = re.search(
                r"Phone:\s(\d{3}-\d{3}-\d{4})\s\|\s(.+)", phones
            ).groups()
```
And then we can look at the size of the list returned? Seems cleaner than nested `try` blocks. e.g.

```
phone1 = None
phone2 = None
if len(phone_numbers) == 1:
  phone1 = phone_numbers[0]
if len(phone_numbers) > 1:
  phone1 = phone_numbers[0]
  phone2 = phone_numbers[1]
```

Reply:
Could even skip the local variable declaration and add the actual values if they are found...

Reply:
Your feedback led me to rethink how the entire section of details was being grabbed, and [I rewrote that in my most recent commit.](https://github.com/openstates/openstates-scrapers/pull/4233/commits/ed72a5d681efad647a8ed541062edfc62cbf383a#diff-9eae9c00f2832ba4d89c6294f928772d6d554299758b67c70db01777d9283412R19-R40)

This new solution is able to store however many phones get added to that field, while also handling the Legislative Assistant, Media Contact, and Email fields all from the same CSS Selector result. This, with the addition of the custom exemption `class NewDetailFieldEncountered(BaseException)` will help us to catch if any modifications or additions happen with these fields.
- Additional win: in actually looking for fully at the whole section, I realized the previous code totally forgot to store the member's email (was only storing the email for their media contact) so this solution grabs that data too.